src/definition/rooms/index.ts

```diff
@@ -17,3 +17,5 @@ export {
     IPreRoomCreatePrevent,
     IPreRoomDeletePrevent,
 };
+
+export * from './IPreRoomUserJoined';
```

Comment:
Why not follow the previous pattern?

Reply:
This way seems to be clearer for me than the previous one. Following an _iterative_ principle, we could apply this to all cases in the future

Reply:
Ok